libs/sdk-backend-mockingbird/src/recordedBackend/dashboards.ts

```diff
@@ -337,6 +337,26 @@ export class RecordedDashboards implements IWorkspaceDashboardsService {
         });
     }
 
+    public exportDashboardToPresentation(
+        _ref: ObjRef,
+        _format: string,
+        _filters?: FilterContextItem[],
+    ): Promise<IExportResult> {
+        return Promise.resolve({
```

Comment:
This can never work, I would rather throw unsupported error or such (same for the second method in recordedBackend). Or is there any reason to have it there?